postgresql/benchmark.sh

```diff
@@ -1,35 +1,51 @@
 #!/bin/bash
 
+set -eux
+
+PGVERSION=17
+
 sudo apt-get update
-sudo apt-get install -y postgresql-common
-sudo apt-get install -y postgresql-14
+sudo apt install -y postgresql-common
+sudo /usr/share/postgresql-common/pgdg/apt.postgresql.org.sh
+
+sudo apt-get update
+sudo apt-get upgrade
+sudo apt-get install -y postgresql-$PGVERSION
+
+memory=$(awk '/MemTotal/ {print $2}' /proc/meminfo)
```

Comment:
For my understanding: Are the parameters in l. 15-20 best practices / commonplace? Perhaps add a comment to some reference (Stackoverflow?) which recommends them?

Reply:
Lot's of people have their own exact preferences, and the best ofcourse depends on the workload. But it's commonly agreed upon that the defaults are terrible. I didn't spend much time on finding the best, I mainly kept them close to whatever we already had in the "tuned" postgres. I added some comments about these settings and some links now.